scene/resources/texture.cpp

```diff
@@ -82,12 +82,28 @@ void Texture2D::draw_rect_region(RID p_canvas_item, const Rect2 &p_rect, const R
 }
 
 bool Texture2D::get_rect_region(const Rect2 &p_rect, const Rect2 &p_src_rect, Rect2 &r_rect, Rect2 &r_src_rect) const {
+	bool ret = false;
+	if (GDVIRTUAL_CALL(_get_rect_region, ret, p_rect, p_src_rect, r_rect, r_src_rect)) {
+		return ret;
+	}
 	r_rect = p_rect;
 	r_src_rect = p_src_rect;
 	return true;
 }
 
+Ref<Image> Texture2D::get_image() const {
```

Comment:
Need to remove in the header

Reply:
whoops, did that right the first time too before redoing it